src/components/molecules/NavSearchBar/NavSearchList.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { Key } from "react";
+import { FixedSizeList } from "react-window";
+
+import "./NavSearchList.scss";
+
+import {
+  NavSearchResultProps,
+  NavSearchResult,
+} from "components/molecules/NavSearchBar/NavSearchResult";
+
+// @debt refactor this into .scss or find more ergonomic component with same desired functionality as FixedSizeList
+// values are used here due to quirks of the FixedSizeList component
+// NOTE: keep in sync with the .scss files mentioned in comments
+const propsFor = (length: number) => ({
+  itemCount: length,
+  itemSize: 48, // 2 * $search-result-padding--vertical + $search-result-image--size; @see navSearchBarConstants.scss
+  height:
+    0 === length
+      ? 40 // $search-dropdown-height--empty; @see navSearchBarConstants.scss
+      : Math.min(
+          480, // $search-dropdown-height--max; @see NavSearchBar.scss
+          48 * length // 2 * $search-result-padding--vertical + $search-result-image--size; @see navSearchBarConstants.scss
+        ),
+  width: "18.125rem", // $search-dropdown-width; @see navSearchBarConstants.scss
+});
+
+export type NavSearchListItem = NavSearchResultProps & { key: Key };
+
+export interface NavSearchListProps {
+  items?: NavSearchListItem[];
+}
+
+export const NavSearchList: React.FC<NavSearchListProps> = ({ items = [] }) => {
+  const children = items.map(
```

Comment:
'children' has a very explicit and defined usage in React (for elements nested within the JSX tags of this component), we shouldn't overload that terminology:

```suggestion
  const renderedSearchResults = items.map(
```